lib/game_logic.rb

```diff
@@ -11,6 +10,80 @@ def initialize(player1, player2)
               '7', '8', '9'] # We ignore index 0 for simplicity
   end
 
+  def display_board
+    @field.each_index do |i|
+      if (i % 3).zero?
+        p @field[(i - 2)..i] unless i.zero?
+      end
+    end
+  end
+
+  private
+
+  def get_move(player)
```

Comment:
Although this isn't the objective of this review, I still think this method has too many responsibilities and should be split up [`OPTIONAL`].